Rajitha/Palindrome.js

```diff
@@ -0,0 +1,43 @@
+/***************************************************
+ * Determine whether a given input is a palindrome.
+ ***************************************************/
+const palindrome = inputQuery => {
+  var query = '';
+  var count = 0;
+  
+  //If number convert to string
+  if (typeof inputQuery === "number"){
+    inputQuery = inputQuery.toString();
+  }
+  count = inputQuery.length;
+
+  //Convert string to lower case and remove special characters/empty spaces
+  inputQuery = inputQuery.toLowerCase().replace(/\W|_/g, "");
+  
+  //Iterate and store characters starting from end
+  //Compare final value with input
+  for (var i = count-1; i >= 0; i--){
+    query += inputQuery.charAt(i);
+  }
+  return((query === inputQuery)? true : false);
+}
+
+//Function to call main function and test different cases
+const testPalindrome = (fn, query, expect) => {
+  const result = fn(query);
+  console.log((result === expect)? true : false); 
+}
+
+//Call tester function with test cases and expected result
+testPalindrome(palindrome, 12*21, true);
```

Comment:
don't need to pass the function and input as separate arguments. just call the function with the input values and pass that down.

Reply:
nvm you should pass the arguments but you should show them if it doesnt pass.